src/avkit.cs

```diff
@@ -309,6 +243,48 @@ interface AVPlayerViewControllerDelegate
 		[NoiOS, TV (11,0), NoWatch, NoMac]
 		[Export ("playerViewController:willTransitionToVisibilityOfTransportBar:withAnimationCoordinator:")]
 		void WillTransitionToVisibilityOfTransportBar ([NullAllowed] AVPlayerViewController playerViewController, bool visible, [NullAllowed] IAVPlayerViewControllerAnimationCoordinator coordinator);
+		
+		[iOS (12,0), NoTV, NoWatch, NoMac]
+		[Export ("playerViewController:willBeginFullScreenPresentationWithAnimationCoordinator:")]
+		void WillBeginFullScreenPresentation (AVPlayerViewController playerViewController, UIViewControllerTransitionCoordinator coordinator);
+		
+		[iOS (12,0), NoTV, NoWatch, NoMac]
+		[Export ("playerViewController:willEndFullScreenPresentationWithAnimationCoordinator:")]
+		void WillEndFullScreenPresentation (AVPlayerViewController playerViewController, UIViewControllerTransitionCoordinator coordinator);
+		
+		[TV (13,0)]
+		[Export ("nextChannelInterstitialViewControllerForPlayerViewController:")]
+		UIViewController NextChannelInterstitialViewController (AVPlayerViewController playerViewController);
+		
+		[TV (13,0)]
+		[Export ("playerViewController:skipToNextChannel:")]
+		void SkipToNextChannel (AVPlayerViewController playerViewController, Action<bool> completion);
```

Comment:
What name for the ```EventArgs```?

Reply:
Everywhere there's an `Action` you're gonna need the `[Async]` attribute. See https://docs.microsoft.com/en-us/xamarin/cross-platform/macios/binding/binding-types-reference#asyncattribute

Reply:
No, that's fine here since this is a protocol